### ISSUE TITLE
Example of styling input and img tags?

### ISSUE THREAD

**Issue by amccloud** [petehunt/jsxstyle#7]
I might be missing something but is this possible with jsxstyle?

**Comment by chcokr** [comment]
I think the point is to include the styling into the parent `<Display>` element (e.g. `<Block>`, `<Flex>`). If you don't want to specify positioning info, then I guess you could still use the `style` attribute within the `<img>`?

**Comment by jrunning** [comment]
Via some trial and error I discovered that you can pass a tag name as the `component` prop and props as the `props` prop. However, assigning other props (e.g. `src`) works, but proves a little wonky:

```jsx
let imgProps = { src: "foo.gif" };
let img = <Inline component="img"
            width="100"
            height="100"
            props={imgProps} />;

React.render(img, document.body);
// => <img src="foo.gif" style="width:100px;height:100px;props:[object Object];display:inline;">
//                                                      wut ---^
```

I'm not sure this behavior is intentional, but it would nice if `props` was blacklisted from the styles.

**Comment by petehunt** [comment]
merged the fix for the bug, more discussion in #32
